app/lib/actions/pulp/abstract_async_task.rb

```diff
@@ -79,7 +79,7 @@ def external_task
         output[:pulp_tasks]
       end
 
-      def cancel!
+      def cancel!(raise_error = true)
```

Comment:
I have to admit - I'm still confused. As I don't see an error within this function that would get raised? Does suspend raise an error?